modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -151,7 +151,7 @@ public synchronized Connection connect(String url, final Properties info) throws
      */
     private Connection wrapConnection(final Connection connection, final JdbcDatabaseContainer container, final ConnectionUrl connectionUrl) {
 
-        final boolean isDaemon = connectionUrl.isInDaemonMode();
+        final boolean isDaemon = connectionUrl.isInDaemonMode() || connectionUrl.isReusable();
```

Comment:
not sure if it should be implicit or explicit, went with implicit for now

Reply:
How would explicitly look in this case?
For me it only makes sense for it  to be implicit.

Reply:
implicit: `jdbc:tc://mysql///?TC_REUSABLE=true`
explicit: `jdbc:tc://mysql///?TC_REUSABLE=true&TC_DAEMON=true` (thrown an error if `TC_REUSABLE` but not `TC_DAEMON`)

Reply:
prefer implicit